### PR TITLE
Remove hack and update report management

### PR DESCRIPTION
**Pull request by ayolab**

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/modifications/CompositeModificationsTest.java
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/modification/server/modifications/CompositeModificationsTest.java:1]: Not sure it should be here :grin:

File: src/test/java/org/gridsuite/modification/server/service/BuildTest.java
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/modification/server/service/BuildTest.java:1]: Same !

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: src/test/java/org/gridsuite/modification/server/modifications/CompositeModificationsTest.java
Comment (ayolab) [reply src/test/java/org/gridsuite/modification/server/modifications/CompositeModificationsTest.java:1]: I hadn't finished fixing the tests yet :wink:

**Review by dbraquart — COMMENTED** [review]
code: 1 remark
tests : on going

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**